rust/kcl-lib/src/std/args.rs

```diff
@@ -184,7 +184,7 @@ impl Args {
         T::from_kcl_val(&arg.value).ok_or_else(|| {
             let expected_type_name = tynm::type_name::<T>();
             let actual_type_name = arg.value.human_friendly_type();
-            let msg_base = format!("This function expected this argument to be of type {expected_type_name} but it's actually of type {actual_type_name}");
+            let msg_base = format!("This function expected the unlabeled argument to be of type {expected_type_name} but it's actually of type {actual_type_name}");
```

Comment:
Could we use 'input argument' rather than unlabelled? I feel like most of the time users will use a pipeline and then unlabelled doesn't make much sense. We should then start using input arg to mean the magic unlabelled one

Reply:
Done. It would be nice if we changed error messages based on whether it was using a pipeline value, which we know, and specifically mention it. Maybe we can do a pass improving stuff like that post-1.0.